fuzzing/extract_text_fuzzer.py

```diff
@@ -0,0 +1,45 @@
+import sys
+
+import atheris
+
+from fuzz_helpers import EnhancedFuzzedDataProvider
+
+with atheris.instrument_imports():
+    from pdf_utils import PDFValidator, prepare_pdfminer_fuzzing
+    from pdfminer.high_level import extract_text
+
+from pdfminer.psparser import PSException
+
+
+def TestOneInput(data: bytes):
```

Comment:
What do you think of using snake_case for function names? 

I know the atheris docs use CamelCase for Python code, but the rest of this repo uses snake_case.